evaluations/evaluation_aragog.py

```diff
@@ -67,12 +66,16 @@ def run_basic_rag(doc_store, sample_questions, embedding_model, top_k):
         try:
             response = rag.run(
                 data={"query_embedder": {"text": q}, "prompt_builder": {"question": q}, "answer_builder": {"query": q}})
-            predicted_answers.append(response["answer_builder"]["answers"][0].data)
+            predicted_answer = response["answer_builder"]["answers"][0].data
+            if predicted_answer.lower() in ["none", "none."]:
+                predicted_answers.append(None)
```

Comment:
@davidsbatista what do you think of doing this?

Reply:
no comment = OK